packages/tasks/src/model-libraries.ts

```diff
@@ -373,6 +373,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 			wildcard: { path: "*.sentis" },
 		},
 	},
+	"whisperkit": {
```

Comment:
```suggestion
	whisperkit: {
```

`prettier` insists on not using `"` unless necessary.